gosec panics while scannig the project with nil pointer dereference

### Summary
 
While running `gosec` on the project the tool fails with panic. The stack trace is:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1231d3e]

goroutine 1 [running]:
go/token.(*File).Name(...)
        /usr/local/opt/go/libexec/src/go/token/position.go:113
github.com/securego/gosec.(*Analyzer).Check(0xc000164b40, 0xc0017eeb40)
        /Users/c0rwin/golang/pkg/mod/github.com/securego/gosec@v0.0.0-20200401082031-e946c8c39989/analyzer.go:202 +0x11e
github.com/securego/gosec.(*Analyzer).Process(0xc000164b40, {0x0, 0xc0000ae0b8, 0xc0010b4000}, {0xc0010b9000, 0x4ac, 0x3a})
        /Users/c0rwin/golang/pkg/mod/github.com/securego/gosec@v0.0.0-20200401082031-e946c8c39989/analyzer.go:140 +0x169
main.main()
        /Users/c0rwin/golang/pkg/mod/github.com/securego/gosec@v0.0.0-20200401082031-e946c8c39989/cmd/gosec/main.go:329 +0x7c5
```

gosec is compiled from the latest.

--- a/analyzer.go
+++ b/analyzer.go
@@ -219,7 +219,12 @@ func (gosec *Analyzer) load(pkgPath string, conf *packages.Config) ([]*packages.
 func (gosec *Analyzer) Check(pkg *packages.Package) {
 	gosec.logger.Println("Checking package:", pkg.Name)
 	for _, file := range pkg.Syntax {
-		checkedFile := pkg.Fset.File(file.Pos()).Name()
+		fp := pkg.Fset.File(file.Pos())
+		if fp == nil {
+			// skip files which cannot be located
+			continue
+		}
+		checkedFile := fp.Name()
 		// Skip the no-Go file from analysis (e.g. a Cgo files is expanded in 3 different files
 		// stored in the cache which do not need to by analyzed)
 		if filepath.Ext(checkedFile) != ".go" {

